### PR TITLE
Daiji first phase boss battle

### PR DESCRIPTION
**Pull request by contingency40**
* First phase lead-up dialogue
* Dialogue between each form
* First form's three loss scenes
* Second form's loss scene
* Third form's loss scene
* Fourth form's two loss scenes
* First phase post-battle dialogue
* Second phase lead-up dialogue

### CODE REVIEW

File: Gaijinizer patch/CommonEvents_469.txt [hunk 1]
```diff
@@ -17,9 +19,9 @@ Oh...♡. Oh...♡♡#MTLed
 > CONTEXT: CommonEvents_469/124/Dialogue
 > CONTEXT: CommonEvents_521/8/Dialogue
 > CONTEXT: Troops/263/pages/1/59/Dialogue
-\\C[2]Because of Puppet, I don't listen to what the body says
-...!! \\N[1]'s face is before my eyes. I will be drawn to
-the heart of Daiji ...!!#MTLed
+\\C[2]Because of her "Puppet" ability, \\N[1]'s body
+has stopped listening to him!! Before he realizes it, his
+face is being drawn toward Daiji's chest...!!#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_469.txt:25]: Note

File: Gaijinizer patch/CommonEvents_469.txt [hunk 2]
```diff
@@ -40,30 +44,34 @@ keep his sanity in front of the wolf.#MTLed
...
 
 時間をかけて、ゆっくりと我が物にしてくれる。
 > CONTEXT: CommonEvents_469/159/Dialogue/enemy17
-Hmm!!Well...Take your time and slowly make it
-your own.#MTLed
+Hmph!! That's fine...
+
+I'll let you become my plaything in your own
+time.#RonLocalization
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_469.txt:75]: Note

File: Gaijinizer patch/CommonEvents_469.txt [hunk 3]
```diff
@@ -40,30 +44,34 @@ keep his sanity in front of the wolf.#MTLed
...
-, ぃ...ぃ...♡#MTLed
+O-O-k-kay... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 妲己は、距離を離し、戦闘態勢に入った…！
 > CONTEXT: CommonEvents_469/157/Dialogue
-Daiji got away and entered Combat Stance ...!#MTLed
+Daiji has moved back and entered a combat stance...!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_469.txt:64]: Remove "has" here.

File: Gaijinizer patch/CommonEvents_472.txt [hunk 4]
```diff
@@ -4,157 +4,177 @@
...
 親友の仇に、気持ちよく中出しした気分はどうじゃ？
 > CONTEXT: CommonEvents_472/3/Dialogue/enemy17
-Hojo, you're right Buzama...
-How is the feeling of having a creampie
-comfortable for your close friend?#MTLed
+Hohoho, so unslightly...
+
+How does it feel to be pleasured by and ejaculate
+inside your foe?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_472.txt:11]: "unslightly" -> "unsightly"

File: Gaijinizer patch/CommonEvents_472.txt [hunk 5]
```diff
@@ -4,157 +4,177 @@
...
 
 妾の妖術で、お前は今から妾の腰を振るだけの人形と化した。
 > CONTEXT: CommonEvents_472/68/Dialogue/enemy17
-Well, let's get rid of power. 
-You are now a puppet with only Hip Thrust dolls.#MTLed
+You can let all the strength out of your body.
+
+With my black magic, you're now nothing more than
+my little thrusting doll.#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_472.txt:100]: Note

File: Gaijinizer patch/CommonEvents_474.txt [hunk 6]
```diff
@@ -101,5 +103,5 @@ Ha ha ha ha ha#MTLed
 > BEGIN STRING
 グルルルルルルル……
 > CONTEXT: CommonEvents_474/295/Dialogue/enemy18
-* purr *#MTLed
+Grrrrrr#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_474.txt:107]: add some ellipses here...

File: Gaijinizer patch/CommonEvents_483.txt [hunk 7]
```diff
@@ -26,8 +27,8 @@ here ...#MTLed
 その身にたっぷりと、思い知らせてやろう……
 > CONTEXT: CommonEvents_483/24/Dialogue/enemy17
 > CONTEXT: CommonEvents_572/24/Dialogue/enemy17
-The samurai was called the jealousy of the
-country, "Regen"...I will let you know in full#MTLed
+I will show you why they called me the
+beautiful kitsune, "Yuen".#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_483.txt:32]: Note

File: Gaijinizer patch/CommonEvents_483.txt [hunk 8]
```diff
@@ -36,8 +37,7 @@ country, "Regen"...I will let you know in full#MTLed
  妲己の姿が植物の悪魔のようになっていた……！！
 > CONTEXT: CommonEvents_483/50/Dialogue
 > CONTEXT: CommonEvents_572/50/Dialogue
-When I appeared next. The figure of Daiji was like a plant
-demon ......!!#MTLed
+Daiji has now taken the form of a plant demon!!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_483.txt:41]: Note

File: Gaijinizer patch/CommonEvents_483.txt [hunk 9]
```diff
@@ -46,8 +46,8 @@ demon ......!!#MTLed
 妖狐の十八番、\\C[2]『変化の術』\\C[0]を見るのは初めてか？
 > CONTEXT: CommonEvents_483/54/Dialogue/enemy18
 > CONTEXT: CommonEvents_572/54/Dialogue/enemy18
-Hihihi ...... Eighteenth of the Insult,\\C[2] "The art
-of change"\\C[0] Is this your first time to see?#MTLed
+Fufufu... Eighteenth of the Insult,\\C[2] "The art
+of change"\\C[0] Is this your first time to see?#CheckLater
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_483.txt:51]: Note

File: Gaijinizer patch/CommonEvents_485.txt [hunk 10]
```diff
@@ -85,32 +88,35 @@ I'm pulled by the ivy, and the distance gradually shrinks
 
 妾の花粉、たっぷりと味わうがよい…！♡
 > CONTEXT: CommonEvents_485/109/Dialogue/enemy18
-Hoh ho...It's late!!Pollen of persimmon,
-please taste plenty ...! Moth#MTLed
+Hohoho... Too slow!!
+
+Go on, have plenty of my pollen! ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_485.txt:94]: Note

File: Gaijinizer patch/CommonEvents_490.txt [hunk 11]
```diff
@@ -4,17 +4,20 @@
...
 
 いや、養分…と呼んだ方がいいかな？
 > CONTEXT: CommonEvents_490/6/Dialogue/enemy18
-Hehehe .... Now that you are a nephew ...
-No, what should I call it nourishment ...?#MTLed
+Kukuku... You're all mine now...
+
+Perhaps I should call you my "nutrients" from
+now on...#MTLed
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_490.txt:21]: Don't forget to add your credits here.

File: Gaijinizer patch/CommonEvents_490.txt [hunk 12]
```diff
@@ -95,152 +103,170 @@ This does not have to do "punishment" .........#MTLed
...
 > BEGIN STRING
 ん…………♡♡
 
 ぁ………ぁぁ……♡♡
 > CONTEXT: CommonEvents_490/195/Dialogue/kazuya-6
-Hm ............ Oh......... oh...♡♡#MTLed
+Mnnph... ♡♡
+
+O-... Okay... ♡♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_490.txt:152]: note

File: Gaijinizer patch/CommonEvents_490.txt [hunk 13]
```diff
@@ -95,152 +103,170 @@ This does not have to do "punishment" .........#MTLed
...
+
+Before finally gobbling you up... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 最高の快楽と共に、ゆっくりと眠れ…♡
 > CONTEXT: CommonEvents_490/349/Dialogue/enemy18
-Sleep slowly with the best pleasures...♡#MTLed
+Sleep well with the greatest pleasure... ♡#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_490.txt:272]: Note

File: Gaijinizer patch/CommonEvents_494.txt [hunk 14]
```diff
@@ -24,104 +26,113 @@ Arooooo!!#Triketos
...
+This f-feels so good... ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 クゥン……♡
 > CONTEXT: CommonEvents_494/46/Dialogue/enemy18
 > CONTEXT: CommonEvents_494/163/Dialogue/enemy18
-*excited whine* ♡#Ron
+*excited whine* ♡#CheckLater
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_494.txt:69]: Note

File: Gaijinizer patch/CommonEvents_496.txt [hunk 15]
```diff
@@ -64,8 +70,10 @@ quality"...How can she change her?#MTLed
 ケダモノのような激しい交尾はお嫌いだったかしら……。
 > CONTEXT: CommonEvents_496/53/Dialogue/enemy18
 > CONTEXT: CommonEvents_574/43/Dialogue/enemy18
-...♪So struggling so hard...
-I hated intense mating like a beard ....#MTLed
+Kusu... ♪
+
+You struggled so desperately, perhaps you do not
+enjoy rough, bestial mating...#MTLed
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_496.txt:77]: Credits here.

File: Gaijinizer patch/CommonEvents_501.txt [hunk 16]
```diff
@@ -41,8 +46,10 @@ mmmphーーーー!!mmmph ん!!! !#MTLed
 せっかくのダンスの旋律が乱れてしまいます…♡
 > CONTEXT: CommonEvents_501/71/Dialogue/enemy18
 > CONTEXT: CommonEvents_525/75/Dialogue/enemy18
-See. Please be quiet……. The melody of the dance
-is disturbed...♡#MTLed
+Shhhhhhh...
+
+Quiet now... You're disturbing the dance
+melody... ♡#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_501.txt:53]: "dance melody" -> "dance's melody"

File: Gaijinizer patch/CommonEvents_506.txt [hunk 17]
```diff
@@ -4,7 +4,7 @@
 > CONTEXT: CommonEvents_506/25/Dialogue
 > CONTEXT: Troops/241/pages/20/39/Dialogue
 > CONTEXT: Troops/242/pages/1/37/Dialogue
-\\N[1] was in close contact with Daiji and got fucked!!#MTLed
+\\N[1] is being raped while glued to Daiji!!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_506.txt:8]: Note

File: Gaijinizer patch/CommonEvents_507.txt [hunk 18]
```diff
@@ -4,26 +4,26 @@
...
 > BEGIN STRING
 \\N[1]の知能から…
 
 \\C[2]『精神統一の仕方』\\C[0]が吸いとられてしまった……
 > CONTEXT: CommonEvents_507/20/Dialogue
-From \\N[1]'s Intelligence ... \\C[2]"The way of concentrate"\\C[0] Has
-been sucked ...#MTLed
+\\C[2]Knowledge of how to use the concentrate skill has
+been sucked out of \\N[1]'s mind...\\C[0]#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_507.txt:18]: Note (check the rest of this file later).

File: Gaijinizer patch/CommonEvents_520.txt [hunk 19]
```diff
@@ -20,8 +22,9 @@ ahhh......... ahhh............#MTLed
 そんなにボロボロになって這いつくばって……
 > CONTEXT: CommonEvents_520/23/Dialogue/enemy18
 > CONTEXT: CommonEvents_575/23/Dialogue/enemy18
-oh no, it's very ...... It's so smashed so much
-...#MTLed
+Oh dear, how troubling...
+
+You're all worn out on your hands and knees...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_520.txt:28]: Note

File: Gaijinizer patch/Map175.txt [hunk 20]
```diff
@@ -36,54 +36,60 @@ Yuuma's Soul!!#MTLed
...
  妾をここまでコケにした人間は初めてじゃ…！
 > CONTEXT: Map175/events/2/pages/0/41/Dialogue/enemy17
-Have you finally grown up ...?!!
-This is the first time a person who has made a
-moss so far ...!#MTLed
+Have you finally become obedient...?!!
+
+This is the first time I've been humiliated by
+a human...!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map175.txt:65]: Note

File: Gaijinizer patch/Map172.txt [hunk 21]
```diff
@@ -185,18 +185,17 @@ Very well, I'll reveal the truth to you, \\N[1].#Triketos
...
 > CONTEXT: Map172/events/2/pages/0/120/Dialogue
 > CONTEXT: Map172/events/2/pages/1/296/Dialogue
-Before my eyes, Yuuma's appearance was 
-cast off... In the blink of an eye, it 
-transformed into a woman that looked like a
-demonic fox...!!#Triketos
+Yuuma's appearance changed in the blink of an eye...
+
+Transforming into a fox-like female demon!!#Triketos/Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:191]: Note

File: Gaijinizer patch/Map172.txt [hunk 22]
```diff
@@ -381,41 +390,44 @@ person as it is.#MTLed
...
 
 > BEGIN STRING
 \\{\\{その姿を！！
  騙るな！！
 > CONTEXT: Map172/events/2/pages/0/220/Dialogue/kazuya-b
-\\{\\{That figure!!Don't scold me!
-!#MTLed
+\\{\\{Don't you dare take his
+form!!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:424]: Note

File: Gaijinizer patch/Map172.txt [hunk 23]
```diff
@@ -573,60 +592,62 @@ I will not let you do what you want.
...
 \\C[10]認証コード！レッドアラート！！
 
 シーケンス起動！！！
 > CONTEXT: Map172/events/2/pages/0/344/Dialogue/yuuma
-\\C[10]authentication code!Red alert! !
-Sequence start!!!#MTLed
+\\C[10]Authentication code: Red alert!!
+
+Sequence start!!!#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:624]: Note

File: Gaijinizer patch/Map172.txt [hunk 24]
```diff
@@ -573,60 +592,62 @@ I will not let you do what you want.
...
 
 > BEGIN STRING
 施設アナウンス
 　『認証コード、確認』
 > CONTEXT: Map172/events/2/pages/0/350/Dialogue
-Facility announcement. "Authentication code, confirmation"#MTLed
+Facility announcement: "Authentication code accepted.".#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:631]: Note

File: Gaijinizer patch/CommonEvents_483.txt [hunk 25]
```diff
@@ -15,9 +16,9 @@ There is nothing but survival to here.#MTLed
 おヌシがここで命尽きる運命に変わりはない……
 > CONTEXT: CommonEvents_483/20/Dialogue/enemy17
 > CONTEXT: CommonEvents_572/20/Dialogue/enemy17
-Well, now we're just looking at it ...
-There is no change to the fate that Nushi dies
-here ...#MTLed
+However, this was just a test of your ability...
+
+Your destiny, to die here, has not changed...#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_483.txt:22]: Remove all commas in the second sentence.

File: Gaijinizer patch/CommonEvents_520.txt [hunk 26]
```diff
@@ -90,33 +93,36 @@ hungry#MTLed
...
 この状況を打破できる『行動』は………！？
 > CONTEXT: CommonEvents_520/102/Dialogue
-\\C[2]Think about it...... there is nothing ...
-The "action" that can overcome this situation is .........!
-?#MTLed
+\\C[2]Think... Is there nothing I can do...?
+
+There has to be some "action" I can take to get out of
+this situation...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_520.txt:128]: Capitalize  "Action" here.

File: Gaijinizer patch/Map172.txt [hunk 27]
```diff
@@ -381,41 +390,44 @@ person as it is.#MTLed
 ミカを殺した時ですら、全て『ユウマ』の本心じゃ
 > CONTEXT: Map172/events/2/pages/0/204/Dialogue/enemy17
 > CONTEXT: Map172/events/2/pages/1/377/Dialogue/enemy17
-\\C[2]Words of concern for you...Encouragement ...
-Anhui...Even when you killed Micah, it's all
-about Yuuma#MTLed
+\\C[2]Those times I worried about you, encouraged you,
+reassured you, even when I killed Micah, those
+were all "Yuuma's" true feelings.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:396]: minor flow:
>_**Those times I worried about you... encouraged
you... reassured you... Even when I killed Micah,
those were all "Yuuma's" true feelings.**_

File: Gaijinizer patch/Map172.txt [hunk 28]
```diff
@@ -381,41 +390,44 @@ person as it is.#MTLed
...
 だから、幼馴染みの親友である、お前にも見抜けなかった
 
  そうだろう、\\N[1]？
 > CONTEXT: Map172/events/2/pages/0/211/Dialogue/yuuma
-So I was a close friend of my childhood, I did
-not see you. That's right, \\N[1]?#MTLed
+So even you, my dearest childhood friend,
+couldn't see through me. Isn't that right,
+\\N[1]?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:406]: Line space between first and second sentences.

File: Gaijinizer patch/CommonEvents_507.txt [hunk 29]
```diff
@@ -42,15 +42,15 @@ From \\N[1]'s Intelligence ... \\C[2]"How to endure"
 \\C[2]『自分を叩く・殴る意味』
 『もがき方、暴れ方』\\C[0]が吸いとられてしまった……
 > CONTEXT: CommonEvents_507/49/Dialogue
-From \\N[1]'s Intelligence ... \\C[2]"The meaning of hitting and
-beating yourself". "Memo, Rampage"\\C[0] Has been sucked ...#MTLed
+\\C[2]Face punch, face slap, struggle, and rampage have been
+sucked out of \\N[1]'s mind...\\C[0]#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_507.txt:47]: minor rewording here, similar to the previous examples:
>_**\\\C[2]"Face Punch", "Face Slap", "Struggle", and "Violent Rage" have
been sucked out of \\\N[1]'s mind...\\\C[0]**_

File: Gaijinizer patch/Map172.txt [hunk 30]
```diff
@@ -635,35 +656,35 @@ I thought that I was looking at all of my
...
 > CONTEXT: Map172/events/2/pages/0/368/Dialogue/yuuma
-\\C[2]Believing that I gave in
-I did not doubt that there was a "memory" that I
-did not release. \\C[0]This is...my favorite 'back
-hand'! !#MTLed
+\\C[2]You thought I'd given in, you didn't consider
+the possibility I'd held back any memories.\\C[0]
+
+This is the ace up my sleeve!!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map172.txt:663]: Add "but" before "you didn't".

File: Gaijinizer patch/CommonEvents_472.txt (hunk 4 above)
Comment (contingency40) [reply Gaijinizer patch/CommonEvents_472.txt:11]: Found another "unslightly" all the way back in Mukuro's dialogue, I'll fix that while I'm at it.

**Pull request merged**